src/extra/plugin/remote.js

```diff
@@ -1,271 +1,104 @@
-// `window.ParsleyExtend`, like `ParsleyAbstract`, is inherited by `ParsleyField` and `ParsleyForm`
-// That way, we could add new methods or redefine some for these both classes. In particular case
-// We are adding async validation methods that returns promises, bind them properly to triggered
-// Events like onkeyup when field is invalid or on form submit. These validation methods adds an
-// Extra `remote` validator which could not be simply added like other `ParsleyExtra` validators
-// Because returns promises instead of booleans.
-(function($){
-
-window.ParsleyExtend = window.ParsleyExtend || {};
-window.ParsleyExtend = $.extend(true, window.ParsleyExtend, {
-  asyncSupport: true,
-
-  asyncValidators: {
-    'default': {
-      fn: function (xhr) {
-        return 'resolved' === xhr.state();
+(function ($) {
+
+  $.extend(true, window.Parsley, {
+    asyncValidators: {
+      default: {
```

Comment:
I think leaving `default` not in a string form could really mess with IE8 as it is a [reserved word](https://mathiasbynens.be/notes/reserved-keywords).